spec/models/text_lesson_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe TextLesson, type: :model do
+  it 'attributes name and video_code cannot be blank' do
+    text_lesson = TextLesson.new
+
+    expect(text_lesson).to_not be_valid
+    expect(text_lesson.errors[:name]).to include('não pode ficar em branco')
+    expect(text_lesson.errors[:lesson_body]).to include('não pode ficar em branco')
+  end
+
+  it 'belongs to course' do
+    text_lesson = TextLesson.new
+
+    expect(text_lesson).to_not be_valid
+    expect(text_lesson.errors[:course]).to include('é obrigatório(a)')
+  end
```

Comment:
Como o projeto tem a gem shoulda-matchers todos esses testes poderiam virar:

```suggestion
  it { should validate_presence_of(:name).with_message('não pode ficar em branco') }
  it { should validate_presence_of(:lesson_body).with_message('não pode ficar em branco') }
  # esse não lembro se funciona pro belongs_to
  it { should validate_presence_of(:lesson_body).with_message('é obrigatório(a)') }
  it { should belong_to(:course) }
```